app.py

```diff
@@ -125,8 +125,7 @@ def cli(verbose, solver_results_store_host, analysis_results_store_host, graph_h
     solver_store.connect()
 
     with _METRIC_SECONDS.time():
-        _LOGGER.info(
-            f"Syncing solver results from {solver_results_store_host} to {graph_hosts}")
+        _LOGGER.info(f"Syncing solver results from {solver_store.ceph.host} to {graph.hosts}")
```

Comment:
please add the document name/id too :+1:

Reply:
This is already present on line bellow (same for analysis docs).

Reply:
Would it be easier to have it in one log line?

Reply:
Ja, we can do it (wanted to avoid it not to have too verbose logs). So you want something like:
```python
_LOGGER.info(f"Syncing solver {document_id} result from {solver_store.ceph.host} to {graph.hosts}")
```

Reply:
Ja I think this way it could be easy to trace a doc thru the whole system

Reply:
Adjusted.